docs/Tutorials/modifyUserAuth.md

```diff
@@ -1,13 +1,37 @@
 --8<-- "References/abbreviations.md"
 
 
-As part of the Open AMT Cloud Toolkit reference implementation, MPS and the Kong service issue and authenticate a JSON Web Token (JWT) for user authentication. The default configuration offers authentication functionality, but it does not support many common configuration options, such as user groups. In a production environment, alternative authentication is available in 0Auth 2*, Lightweight Directory Access Protocol (LDAP), Kerberos*, etc.
-
-!!! Warning
-    In the current release, if you choose to modify the toolkit's default authentication, no keyboard, video and mouse (KVM) or serial over LAN (SOL) support will be available. 
+As part of the Open AMT Cloud Toolkit reference implementation, MPS and the Kong service issue and authenticate a JSON Web Token (JWT) for user authentication. The default configuration offers authentication functionality, but it does not support many common configuration options, such as user groups. In a production environment, alternative authentication is available in 0Auth 2*, Lightweight Directory Access Protocol (LDAP), Kerberos*, and more.
 
 The instructions below explain how to add an LDAP plugin to Kong.
 
+## User Authentication Flow
+
+The following diagram helps illustrate the typical user authentication flow. Learn more about the `authorize` API call in the [REST API Call Tutorial](./apiTutorial.md) or directly in our [API Documentation](../APIs/indexMPS.md).
+
+<figure class="figure-image">
+<img src="..\..\assets\images\UserAuth_Diagram.png" alt="Figure 1: User Authentication Flow">
+<figcaption>Figure 1: User Authentication Flow</figcaption>
+</figure>
+
+### To authenticate for REST APIs:
+
+1. Call the MPS REST API `/api/v1/authorize` to receive an authentication token from the User Auth Service.
+
+### To authenticate for Websocket connections:
+
+1. Call the MPS REST API `/api/v1/authorize` to receive an authentication token from the User Auth Service.
```

Comment:
The diagram seems to indicate this call is to the User Auth Service.  Later on we mention a 3rd party auth service can be used.  it seems like the diagram and the flow steps need to work together better?

Reply:
Seems wierd to me that we list out the same step again -- feels like it would be better to say something like "Using the auth token from step 1, call the rest api .... etc. Right now, to me , it reads that i need to call api/v1/authorize twice, once for the rest apis and once for the websocket.

Reply:
I've reworked the page a bit with an additional diagram.  My intention was that people would read them separately rather than sequentially.  Might be clearer now, let me know what you guys think.